src/modules/endpoint/DefaultDataContainer.js

```diff
@@ -31,133 +64,86 @@ class DefaultDataContainer implements DataContainer {
     return this._cities
   }
 
-  getCategoriesMap = async (): Promise<CategoriesMapModel> => {
-    if (this._context === null) {
-      throw Error('Context has not been set yet.')
-    }
-    if (this._categoriesMap === null) {
-      throw Error('CategoriesMap is null.')
-    }
-    return this._categoriesMap
+  getCategoriesMap = (context: DatabaseContext): Promise<CategoriesMapModel> => {
+    const cache: Cache<Array<EventModel>> = this.caches['categories']
```

Comment:
Why don't you just use `this.caches.categories`? That way flow should be able to check the types.

Reply:
Done

Reply:
Not really done, right? :thinking:

Reply:
Same for other getters

Reply:
It actually is done, couldn't push because Github was down...

Reply:
Oh okay